### ISSUE TITLE
Word.export_to_html() is broken

### ISSUE THREAD

**Issue by vasichkin** [automagica/automagica#115]
Automagica version 2.0.25

```
        wrd = Word(file_path=self.file_path)
        wrd.export_to_html("D:\\temp\\output.html")
```

Cause error:

```
Traceback (most recent call last):
  File "D:/Automagica/flow1/wordParser.py", line 30, in <module>
    print(exc.exctract_name().name)
  File "D:/Automagica/flow1/wordParser.py", line 17, in exctract_name
    wrd.export_to_html("D:\\temp\\output.html")
  File "C:\Users\vkhomenko\AppData\Local\Programs\Python\Python38\lib\site-packages\automagica\utilities.py", line 17, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\vkhomenko\AppData\Local\Programs\Python\Python38\lib\site-packages\automagica\activities.py", line 3334, in export_to_html
    word.app.ActiveDocument.WebOptions.RelyOnCSS = 1
NameError: name 'word' is not defined
```

**Comment by kathawala** [comment]
I think it's a typo.

In other functions like `export_to_pdf` you have

```python
        self.app.ActiveDocument.ExportAsFixedFormat(OutputFileName=path,
                                                    ExportFormat=17,
                                                    OpenAfterExport=False,
                                                    OptimizeFor=0,
                                                    CreateBookmarks=1,
                                                    DocStructureTags=True
                                                    )
```

but in `export_to_html` you have
```python
        word.app.ActiveDocument.WebOptions.RelyOnCSS = 1
        word.app.ActiveDocument.WebOptions.OptimizeForBrowser = 1
        word.app.ActiveDocument.WebOptions.BrowserLevel = 0
        word.app.ActiveDocument.WebOptions.OrganizeInFolder = 0
        word.app.ActiveDocument.WebOptions.UseLongFileNames = 1
        word.app.ActiveDocument.WebOptions.RelyOnVML = 0
        word.app.ActiveDocument.WebOptions.AllowPNG = 1
        word.app.ActiveDocument.SaveAs(
            FileName=path, FileFormat=wc.wdFormatHTML)
```

I think changing `word.app` to `self.app` in the `export_to_html` function should fix it... but I'm not familiar with the codebase...

**Comment by koenvaneijk** [comment]
Hi @vasichkin and @kathawala,

We fixed the issue, this was indeed a typo/copy pasta mistake.

Thanks for letting us know.

You can find the latest version in `master` and this will be included in the 2.1.2 release on PyPI later today.
